communications/blogs/turing_joins_binder_federation.md

```diff
@@ -0,0 +1,102 @@
+
+There are a lot of tools available that can make packaging together code and resources a lot easier.
+[Jupyter Notebooks](https://jupyter-notebook.readthedocs.io/en/stable/notebook.html) are a very popular method of combining code and prose to produce an interactive scientific analysis.
+[Docker](https://docs.docker.com/engine/docker-overview/) is used to bundle together, code, data and the software environment into a "container" in such a way that it is easily reused and portable across different systems.
+Using cloud computing resources, these tools can be made accessible to anyone, anywhere in the world.
+However, learning all of these tools individually (let alone combining them together) can be a huge barrier to researchers since they require a lot of time investment to learn and implement properly.
+
+[BinderHub](https://binderhub.readthedocs.io/en/latest/) is the computational infrastructure that coordinates all of these tools to provide an easy method of creating computing environments for scientific analyses that are reproducible and shareable with anyone, anywhere.
+It rewards reproducible data science and software engineering best practices as it requires a version controlled project with documented software dependencies in order to automatically generate the environment.
+
+Turing Research Fellow and lead developer of _The Turing Way_, [Dr Kirstie Whitaker](https://www.turing.ac.uk/people/researchers/kirstie-whitaker), had the following comment on the BinderHub technology.
+
+> BinderHubs package together really fantastic tools like notebooks, docker, cloud computing resources, coordination engines, along with clear version control to deliver reproducible research at the click of a button.
+>
+> -- Dr Kirstie Whitaker
+
+![BinderHub](https://zenodo.org/api/iiif/v2/e4125eaf-b456-4097-85fc-6a2e80482d1c:96c70193-2f9e-442d-8cf8-21485d8864e1:1728_TURI_Book%20sprint_45%20repo2docker_040619_v2_MK.jpg/full/750,/0/default.jpg)
+
+## The Binder Federation
+
+[The Binder Federation](https://blog.jupyter.org/the-international-binder-federation-4f6235c1537e) is an experiment in open infrastructure for data science and beyond.
+It is a community of scientists and open-source developers dedicated to scientific reproducibility, communicating ideas through interactive computing, and providing real-time access to results and analyses.
+The Binder Federation provides interactive data analytics environments to roughly 10,000 unique repositories via 300,000 launches a month, and it relies on its team of partner BinderHubs to collaborate in providing this service.
+
+GKE, OVH, GESIS and now new members the Turing pool their resources to create a sustainable, open-source infrastructure that allows easy creation, sharing of and access to reproducible computational environments.
```

Comment:
Can we make GkE, OVH and Gesis links people can click?

Reply:
Done in 6119674